File: apps/s-2-admin/src/api/team/TeamCreateInput.ts
import { EventTypeCreateNestedManyWithoutTeamsInput } from "./EventTypeCreateNestedManyWithoutTeamsInput";
import { MembershipCreateNestedManyWithoutTeamsInput } from "./MembershipCreateNestedManyWithoutTeamsInput";

export type TeamCreateInput = {
  name?: string | null;
  slug?: string | null;
  logo?: string | null;
  bio?: string | null;
  hideBranding: boolean;
  eventTypes?: EventTypeCreateNestedManyWithoutTeamsInput;
  members?: MembershipCreateNestedManyWithoutTeamsInput;
};

File: apps/cactus-demo-dotnet/src/APIs/Team/Dtos/TeamCreateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class TeamCreateInput
{
    public int? Id { get; set; }

    public string? Name { get; set; }

    public string? Slug { get; set; }

    public string? Logo { get; set; }

    public string? Bio { get; set; }

    public bool HideBranding { get; set; }

    public List<EventType>? EventTypes { get; set; }

    public List<Membership>? Members { get; set; }
}
